tests/ut/py/test_callable_identity.py

```
@@ -109,6 +399,899 @@ def test_worker_register_returns_opaque_handle_and_deduplicates_same_identity():
+            RemoteWorkerSpec(endpoint=f"127.0.0.1:{port}", platform="a2a3sim", transport="sim")
+        )
+        handle = worker.register(
+            RemoteCallable("tests.ut.py.test_callable_identity:_remote_exit_orch"),
+            workers=[endpoint],
+        )
+        worker.init()
+
+        def parent_orch(orch, _args, cfg):
+            orch.submit_next_level(handle, TaskArgs(), cfg, worker=endpoint)
+
+        with pytest.raises(RuntimeError, match="RemoteL3Endpoint::run|socket closed|health lane"):
```

Ruff (0.15.17)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)